src/backend/controllers/battleController.js

```diff
@@ -174,6 +174,13 @@ exports.reportResults = async (req: Request, res: Response) => {
 					.status(404)
 					.json({ msg: 'Could not find userTwo'});
 			}
+
+			battle.winner = winner;
+
+			// Save battle winner and users money
+			await battle.save();
+			await userOne.save();
+			await userTwo.save();
```

Comment:
You can just save the battle only. `findByIdAndUpdate` saves the users already.

Reply:
Okay, should be fixed now